scaler/main.go

```diff
@@ -50,9 +55,15 @@ func main() {
 	targetPendingRequests := cfg.TargetPendingRequests
 	targetPendingRequestsInterceptor := cfg.TargetPendingRequestsInterceptor
 
-	k8sCl, _, err := k8s.NewClientset()
+	k8sCfg, err := rest.InClusterConfig()
```

Comment:
I know that other components don't use it, but I think that it's better if we do
```suggestion
        // import ctrl "sigs.k8s.io/controller-runtime"
	k8sCfg := ctrl.GetConfigOrDie()
```

Thanks to this change, we can debug the code locally without any change. (and maybe we should modify the other places to use this approach)

Reply:
Great suggestion! I’ll change that.